commands/curation/curationaudit.go

```diff
@@ -259,6 +305,9 @@ func (ca *CurationAuditCommand) doCurateAudit(results map[string][]*PackageStatu
 		if err := ca.auditTree(techutils.Technology(tech), results); err != nil {
 			return err
 		}
+		ca.setPackageManagerConfig(nil)
+		ca.AuditParams = ca.SetDepsRepo("")
```

Comment:
```suggestion
		ca.setPackageManagerConfig(nil)
		ca.AuditParams = ca.SetDepsRepo("")
```
Why are we setting this after after each tech (first one does not have this values, is it ok?) If there is a reason, please add a comment.

Reply:
Its a bug that I figured out during testing the jf ca with multiple working dirs for different package types.
So lets say the first working dir is npm, it will set deps repo of npm, and the second working dir is pip, if the deps repo already filled it doesn't fill it again, so the outcome is that it try to resolve pip project deps from npm repo. Thats why I need to reset it on each iteration so it will set the deps repo corectly.